src/components/Details/EventVideo/EventVideo.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { FC, RefObject, useEffect, useImperativeHandle, useRef } from "react";
+
+import videojs, { VideoJsPlayer } from "video.js";
+
+import { strings } from "../../../assets/LocalizedStrings";
+
+import { initVideoJsLanguages } from "./utils";
+
+import "video.js/dist/video-js.css";
+import "./vjs-theme-cdp.css";
+
+initVideoJsLanguages();
+
+enum KeyBoardKey {
+  SPACE = 32,
+  ARROW_LEFT = 37,
+  ARROW_RIGHT = 39,
+  F = 70,
+  K = 75,
+  M = 77,
+}
+
+const SKIP_SECONDS = 10;
+
+/**Public API with seekTo method */
+export interface EventVideoRef {
+  seekTo(seconds: number): void;
+}
+
+export interface EventVideoProps {
+  /**The source uri of the video */
+  uri: string;
+  /**Event video Transcript reference */
+  componentRef: RefObject<EventVideoRef>;
+  /**The initial seconds to seek to. */
+  initialSeconds?: number;
+}
+
+const EventVideo: FC<EventVideoProps> = ({
+  uri,
+  componentRef,
+  initialSeconds,
+}: EventVideoProps) => {
+  const videoRef = useRef<HTMLVideoElement>(null);
+  const videoJsPlayerRef = useRef<VideoJsPlayer>();
+
+  useImperativeHandle(componentRef, () => ({
+    /**Implement componentRef.seekTo by using videoJsPlayerRef.currentTime and play method */
+    seekTo: (seconds: number) => {
+      videoJsPlayerRef.current?.currentTime(seconds);
+      if (videoJsPlayerRef.current?.paused()) {
+        videoJsPlayerRef.current?.play();
+      }
+    },
+  }));
+  const VideoHtml = () => (
+    <div data-vjs-player>
+      <video ref={videoRef} className="video-js vjs-big-play-centered vjs-theme-cdp" />
+    </div>
+  );
+
+  useEffect(() => {
+    const videoElement = videoRef.current;
+    let player: VideoJsPlayer;
+    if (videoElement) {
+      player = videojs(
+        videoElement,
+        {
+          autoplay: false,
+          controls: true,
+          controlBar: {
+            currentTimeDisplay: true,
+            timeDivider: true,
+            durationDisplay: true,
+            customControlSpacer: true,
+            remainingTimeDisplay: false,
+          },
+          preload: initialSeconds ? "auto" : "metadata",
+          aspectRatio: "16:9",
+          fluid: true,
+          language: strings.getLanguage(),
+          playbackRates: [0.75, 1, 1.5, 2],
```

Comment:
Can we add 0.5 and reverse the order of this list. I am so used to the youtube version where it's the reverse order of this.

Reply:
Yes to both